src/mlbstandings/web.py

```
@@ -3,25 +3,35 @@
+            code = e.code
+        else:
+            return False
+        return not Web.retryable(code)
     @staticmethod
     @backoff.on_exception(
         backoff.expo,
         (requests.exceptions.HTTPError, urllib.error.HTTPError, requests.exceptions.ConnectionError, requests.exceptions.Timeout),
         max_time=600,
-        giveup=lambda e : isinstance(e, requests.exceptions.HTTPError) and e.response.status_code not in set([429, 500, 503]),
+        giveup=lambda e: Web.giveup(e),
         max_value=60
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)